eap-coffee-app/src/main/webapp/index.xhtml

```diff
@@ -14,11 +14,8 @@
 		crossorigin="anonymous" />
 </h:head>
 <h:body>
-    <div class="container text-muted">
-		<p><h:outputFormat value="#{messages.podName}" rendered="#{cafe.hostName != null}">
-			<f:param value="#{cafe.hostName}" />
-		</h:outputFormat></p>
-		<p><h:outputText value="#{messages.versionString}" /></p>
```

Comment:
I think we added this to version string to check if we are deploying the latest version of sample app in [pipeline](https://github.com/Azure-Samples/cargotracker-azure/blob/main/.github/workflows/updateCargoTracker.yml#L78).

I'd suggest we keep this just in case, but I'd also agree to have it removed.

Reply:
Good point. I added `versionString` back in commit 675b003.